README.md

```diff
@@ -16,12 +16,24 @@ npm install -g yo
 npm install -g generator-napi-module
 ```
 
+Create a directory for your N-API module:
+```bash
+mkdir <moduleName>
```

Comment:
You also need to cd into the directory, right?

Reply:
Bah, fair - will update